lib/include/mxl/flow.h

```diff
@@ -346,6 +346,30 @@ extern "C"
      *
      * \param[in] index The head index of the samples to obtain.
      * \param[in] count The number of samples to obtain.
+     * \param[in] timeoutNs How long to wait in nanoseconds for the range of
+     *      samples to become available.
+     * \param[out] payloadBuffersSlices A pointer to a wrapped multi buffer
```

Comment:
(Not specific to this PR)

One `mxlWrappedMultiBufferSlice` represents multiple buffer slices, right?
But all over the place a `mxlWrappedMultiBufferSlice` variable/parameter is called `payloadBuffersSlices` (plural buffers, plural slices)? Just reads a bit odd.